backend/src/binaryInstaller.ts

```diff
@@ -124,7 +120,7 @@ class BinaryInstaller {
       const options = { disableGlob: true }
 
       try {
-        rimraf.sync(installer.binaryPath(), options)
+        rimraf.sync(installer.binaryPathCLI(), options)
```

Comment:
Uninstall should also loop over the installer resource instances and remove the symlinks for each.